server/src/main/java/org/jboss/as/server/controller/git/GitConfigurationPersister.java

```diff
@@ -154,6 +155,8 @@ public String publish(String name) throws ConfigurationPersistenceException {
             } catch (GitAPIException ex) {
                 throw MGMT_OP_LOGGER.failedToPublishConfiguration(ex, name, ex.getMessage());
             }
+        } else {
+            throw MGMT_OP_LOGGER.failedToPublishConfiguration(null, name, INVALID_REMOTE_NAME);
```

Comment:
This should be properly managed in the logger

Reply:
Changed the implementation.